torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -580,8 +580,7 @@ struct GraphFuser {
     // we can move the consumer up into the producer.
     // but this requires better handling of merging fusion groups so it is not
     // done now
-    bool shouldFuse = isFusableDevice(producer) &&
-        isFusable(producer->node()) &&
+    bool shouldFuse = isFusable(producer->node()) &&
```

Comment:
????

Reply:
this `isFusableDevice` check is handled in Bram's previous PR on custom fusion [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/graph_fuser.cpp#L246)

Reply:
Okay, so the huge comment block in the middle of a boolean expression was super confusing. Can we move that out? maybe assign that boolean to its own variable then use that in the boolean expression?

Reply:
this preserves the change/comment block from here:
https://github.com/pytorch/pytorch/commit/5be4bee4ff58d04c8ee73163e2cb2e06bd69b2a5#diff-52ef1c06d69edbd31a705b6d2b827f50

Reply:
basically this diff just fixes a rebase bug